### ISSUE TITLE
More generative model

### ISSUE THREAD

**Issue by davidbenjamin** [davidbenjamin/permutect#247]
Sequencing reads are probably conditionally independent given a hidden covariate.  For example, given that, say, a region is prone to mapping error, the mismatch errors in different reads are independent of each other.  Thus the overall log likelihood ratio (of artifact vs non-artifact or perhaps of specific classes of artifact vs non-artifact) of data ought to be the sum of per-read log-likelihood ratios.  This suggests a simpler architecture where we have some sub-network that computes per-read logits which are then summed to get an overall logit.

If this works, it will 1) have a stronger guarantee of read-count independence, 2) incorporate read counts into statistical confidence naturally, 3) handle the case of no reference reads naturally, 4) permit natural extensions to modeling normal artifacts.

**Comment by davidbenjamin** [comment]
Closed in #256
